components/builder-jobsrv/src/server/scheduler.rs

```diff
@@ -22,15 +22,16 @@ use hab_net::conn::RouteClient;
 use hab_net::socket::DEFAULT_CONTEXT;
 use zmq;
 
-use protocol::jobsrv::{self, Job, JobSpec};
-use protocol::originsrv::*;
-use protocol::scheduler as proto;
+use protocol::jobsrv;
+use protocol::originsrv;
 use data_store::DataStore;
-use error::{SrvResult, SrvError};
```

Comment:
Ugh, I knew not refactoring this in one of the services would eventually bite me :). It's cool to change this all to `Result` but as a pattern I had switched all but the JobSrv to `SrvResult` to clean up the need to `use std::result` and then address a regular result with `result::Result`. If you want to clean it up here by all means, if not we can get it eventually to look like the other servers